src/chaos-experiments/models/database/sequelize/sequelizeConnector.js

```diff
@@ -173,6 +175,23 @@ async function getChaosJobExperimentByJobId(jobId, contextId) {
     return chaosExperiment;
 }
 
+async function getFutureJobExperiments(timestamp, contextId) {
+    const options = {
+        where: {
+            is_triggered: false,
+            start_time: { [Op.gt]: timestamp }
```

Comment:
if possible maybe consider greater or equal, although its not really critical haha